orttraining/orttraining/lazy_tensor/register.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <iostream>
+// Instead of torch/torch.h, include torch torch/extension.h
+// for extra Python headers.
+#include <torch/extension.h>
+#include <torch/csrc/jit/passes/pass_manager.h>
+#include "torch/csrc/jit/passes/shape_analysis.h"
+#include <torch/csrc/jit/runtime/custom_operator.h>
+#include "accelerator.h"
+#include "flags.h"
+#include "fusion.h"
+#include "core/common/logging/logging.h"
+
+namespace onnxruntime {
+namespace lazytensor {
+void register_ort_as_torch_jit_executor() {
+  // Pytorch's JIT symbol to be execute by ORT.
+  const auto accelerator_symbol =
+      torch::jit::Symbol::fromQualString("ort::graph");
```

Comment:
is there any ABI concerns for this api?

Reply:
I think so. This function contains several libtorch APIs.